systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingAbstractST.java

```diff
@@ -171,7 +171,7 @@ void doTestProducerConsumerMirrorMaker2Service(final ExtensionContext extensionC
             .endSpec()
             .build());
 
-        LOGGER.info("Setting for kafka source plain bootstrap:{}", KafkaResources.plainBootstrapAddress(kafkaClusterSourceName));
+        LOGGER.info("Setting for kafka source plain bootstrap: {}", KafkaResources.plainBootstrapAddress(kafkaClusterSourceName));
```

Comment:
these 'kafka' remain in lower case ? I understand that we are referencing config.